invest/templates/invest/high-potential-opportunity-detail.html

```diff
@@ -0,0 +1,231 @@
+                    <img src="{{ page.companies_list_item_image_six.url }}" />
+                    <img src="{{ page.companies_list_item_image_seven.url }}" />
+                    <img src="{{ page.companies_list_item_image_eight.url }}" />
+                </li>
+            </ul>
+        </div>
+    </div>
+</section>
+
+<section id="case-studies">
+    <div class="container">
+        <div class="grid-row">
+            <h2 class="heading-xlarge">{{ page.case_study_list_title }}</h2>
+            <ul>
+                <li>
+                    <div class="column-one-third">
+                        {{ page.case_study_one_text|add_export_elements_classes }}
+                    </div>
+                    <div class="column-two-thirds">
+                        <img src="{{ page.case_study_one_image.url }}" />
+                    </div>
+                </li>
+                {% if page.case_study_two_text %}
+                    <li>
+                        <div class="column-one-third">
+                            {{ page.case_study_two_text|add_export_elements_classes }}
+                        </div>
+                        <div class="column-two-thirds">
+                            <img src="{{ page.case_study_two_image.url }}" />
+                        </div>
+                    </li>
+                {% endif %}
+                {% if page.case_study_three_text %}}
+                    <li>
+                        <div class="column-one-third">
+                            {{ page.case_study_three_text|add_export_elements_classes }}
+                        </div>
+                        <div class="column-two-thirds">
+                            <img src="{{ page.case_study_three_image.url }}" />
+                        </div>
+                    </li>
+                {% endif %}
+                {% if page.case_study_four_text %}
+                    <li>
+                        <div class="column-one-third">
+                            {{ page.case_study_four_text|add_export_elements_classes }}
+                        </div>
+                        <div class="column-two-thirds">
+                            <img src="{{ page.case_study_four_image }}" />
+                        </div>
+                    </li>
+                {% endif %}
+            </ul>
+        </div>
+    </div>
+</section>
+
+<section id="other-opportunities">
+    <div class="container">
+        <h2 class="heading-xlarge">{{ page.other_opportunities_title }}</h2>
+        <div class="grid-row">
+            {% for opportunity in page.other_opportunities %}
+                <div class="column-one-half">
+                    <div>
+                        <img src="{{ opportunity.hero_image.url }}" />
+                        <div class="title-container">
+                            <a class="link" href="{% url 'high-potential-opportunity-details' slug=opportunity.meta.slug %}">{{ opportunity.heading }}</a>
```

Comment:
you can use the cards from directory components here
```
{% url 'high-potential-opportunity-details' slug=opportunity.meta.slug  as opportunity_url %}
{% include 'directory_components/card.html' with img_src=opportunity.hero_image.url url=opportunity_url title=opportunity.heading %}
```
and you can use `card-grid` instead of `grid-row` so they have aligned heights regardless of content 👍 column classes are the same

this needs components v2.22.0

Reply:
had a go, unfortunately, the card component crates one card per row, while we want two cards per row:

### the card
![image](https://user-images.githubusercontent.com/5485798/45370353-f9ea0c80-b5df-11e8-8306-c6b7899e3c31.png)

### but we want

![image](https://user-images.githubusercontent.com/5485798/45370376-053d3800-b5e0-11e8-8655-78c416e7cd36.png)

Reply:
? no you just gotta put them inside a `column` class. they stretch to fit the size of their container

Reply:
there is an example in the components demo page https://github.com/uktrade/directory-components/blob/master/demo/templates/demo/components.html#L108